### PR TITLE
Update dependencies

### LINKED ISSUE #2: Fix range must count the current text length, not the replacement's
I'm implementing a plugin that applies textlint fixes, and noticed that the range is being incorrectly computed.

See the example below:

```
$ echo '2021-07-02 (Thursday)' | textlint --format json --config samples/.textlintrc.json --stdin --stdin-filename file.md | jq
[
  {
    "messages": [
      {
        "type": "lint",
        "ruleId": "date-weekday-mismatch",
        "message": "2021-07-02 (Thursday) mismatch weekday.\n2021-07-02 (Thursday) => 2021-07-02 (Friday)",
        "index": 12,
        "line": 1,
        "column": 13,
        "severity": 2,
        "fix": {
          "range": [
            12,
            18
          ],
          "text": "Friday"
        }
      }
    ],
    "filePath": "<markdown>"
  }
]
```

The range should be within 8 characters (Thursday), instead of 6. If I apply the suggested range, I'll end up with `Fridayay`.

### PR DESCRIPTION
**Pull request by tyler36**
This PR updates package and workflow dependencies.

Fixes #2
Fixes Dependabot CVE-2021-23371

### CODE REVIEW

**Comment by tyler36** [comment]
The tests are not passing locally. I get

```
AssertionError [ERR_ASSERTION]: Could not specify options property in invalid object when TestConfig was passed. Use TestConfig.rules.options.
    at /home/user13/code/pr/textlint-rule-date-weekday-mismatch/node_modules/textlint-tester/src/textlint-tester.ts:410:28
```

File: test/textlint-rule-date-weekday-mismatch-test.js [hunk 1]
```diff
@@ -2,7 +2,16 @@ import TextLintTester from "textlint-tester";
...
+const dateWeekdayMismatchRule = require("../src/textlint-rule-date-weekday-mismatch");
+const rule = {
+    rules: [{
+        ruleId: "date-weekday-mismatch",
+        rule: dateWeekdayMismatchRule,
+        options: {
+            lang: "ja"
+        }
+    }]
+}
```
Comment (azu) [inline test/textlint-rule-date-weekday-mismatch-test.js:14]: Was there a problem with passing the rule object as is?
Most test cases test if no options = automatic determination works well..

**Comment by azu** [comment]
> `AssertionError [ERR_ASSERTION]: Could not specify options property in invalid object when TestConfig was passed. Use TestConfig.rules.options.`

This error is occured when rule's options is conflict.

```
const rule = {
    rules: [{
        ruleId: "date-weekday-mismatch",
        rule: dateWeekdayMismatchRule,
        options: {
            lang: "ja"
        }
    }]
}
```

vs.

```
        {
            text: "2016-12-29(月)",
            output: "2016-12-29(木)",
            options: {
                lang: "ja-JP"
            },
        }
```

File: test/textlint-rule-date-weekday-mismatch-test.js (hunk 1 above)
Comment (azu) [reply test/textlint-rule-date-weekday-mismatch-test.js:14]: `const rule = ...` is for multiple rules or plugin combination.

Comment (tyler36) [reply test/textlint-rule-date-weekday-mismatch-test.js:14]: After updating the dependencies, the tests failed:
```
+ '2016年12月29日(月)'
- '2016年12月29日(木)'
               ^
      + expected - actual

      -2016年12月29日(月)
      +2016年12月29日(木)
```

I assumed this was due to needing the language set?

**Comment by tyler36** [comment]
>  options: { lang: "ja-JP" },

The readme on https://github.com/textlint-rule/textlint-rule-date-weekday-mismatch lists Japanese as `ja` in "Supported lang" and first "Options". Reading more closely, I see `ja-JP` as the second option, but `ja` below that in the ISO 639-1 e.g. I am confused.

File: test/textlint-rule-date-weekday-mismatch-test.js (hunk 1 above)
Comment (tyler36) [reply test/textlint-rule-date-weekday-mismatch-test.js:14]: English also failing:
```
+ '2021-07-02 (Thursday)'
- '2021-07-02 (Friday)'
               ^
      + expected - actual
```

**Comment by azu** [comment]
https://github.com/wanasit/chrono/releases/tag/v2.0.2
> Chrono’s default now handles only international English. While in the previous version, it tried to parse with all known languages.

> // default English (US)
> chrono.parseDate('6/10/2018');   
> https://github.com/wanasit/chrono#locales

chrono v2 seems to have lost the automatic language determination feature.

**Comment by azu** [comment]
> options: { lang: "ja-JP" },

I think it is intentionally vague. But I don't remember why.
Either way, it is not a problem to change the `lang` option since the BREAKING CHANGE is included.

**Comment by tyler36** [comment]
OK ... so I can see this is working.
I am not sure how to get the locale string into this function. 

```javascript
const text = "2016年12月29日(月)"
const chronoDates = chrono.parse(text);
// []

const chronoDates = chrono['ja'].parse(text);
// 0: ParsingResult {text: '2016年12月29日' …}
```

**Comment by tyler36** [comment]
Closing. Stale.

**Pull request closed without merging**